src/Qir/Runtime/lib/QSharpFoundation/qsharp-foundation-qis.ll

```diff
@@ -202,12 +215,8 @@ define dllexport double @__quantum__qis__drawrandomdouble__body(double %min, dou
 
 
 ;===============================================================================
-; quantum.qis conditional functions
+; quantum.qis conditional functions implementation
```

Comment:
In Q#, these are operations rather than functions; does "function" here refer to the Q# signature, or to the projection of that into QIR?

Reply:
Changed to 
> quantum.qis conditional functions/**operations** implementation